chia-tools/Cargo.toml

```diff
@@ -9,14 +9,14 @@ homepage = "https://github.com/Chia-Network/chia_rs/chia-tools"
 repository = "https://github.com/Chia-Network/chia_rs/chia-tools"
 
 [dependencies]
-chia-protocol = { version = "0.5.0", path = "../chia-protocol" }
-chia-traits = { path = "../chia-traits" }
-clvm-utils = { path = "../clvm-utils" }
-clvm-traits = { version = "=0.3.3", path = "../clvm-traits" }
+chia-protocol = { version = "=0.5.0", path = "../chia-protocol" }
+chia-traits = { version = "=0.5.0", path = "../chia-traits" }
+clvm-utils = { version = "=0.5.0", path = "../clvm-utils" }
```

Comment:
I thought we were moving away from pinning versions, other than top level projects (say, the wheel). but maybe if we're moving towards `cargo workspace` it won't matter much

Reply:
Technically it doesn't really matter here since you only run it locally and this never gets published. I just made them consistent with the other dependencies in chia-tools. I think it'd be fine to unpin these as well though especially since the version number doesn't affect much.